synapse_data_warehouse/synapse/tables/V2.19.4__add_userprofile_latest_comments.sql

```diff
@@ -0,0 +1,18 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+COMMENT ON TABLE USERPROFILE_LATEST IS 'This table contain snapshots of user-profiles. Snapshots are taken when user profiles are created or modified. Note: Snapshots are also taken periodically and independently of the changes. The snapshot_timestamp records when the snapshot was taken.';  
```

Comment:
@rxu17 See this: https://github.com/Sage-Bionetworks/snowflake/pull/48#discussion_r1628427498

Reply:
Done